tests/migration_test.go

```diff
@@ -112,11 +114,11 @@ import (
 )
 
 const (
-	fedoraVMSize         = "256M"
-	secretDiskSerial     = "D23YZ9W6WA5DJ487"
-	stressDefaultVMSize  = "100"
-	stressLargeVMSize    = "400"
-	stressDefaultTimeout = 1600
+	fedoraVMSize               = "256M"
+	secretDiskSerial           = "D23YZ9W6WA5DJ487"
+	stressDefaultVMSize        = "100"
+	stressLargeVMSize          = "400M"
```

Comment:
You should adjust the `stressDefaultVMSize` as well, since it is used as parameter for  `runStressTest` function.
```suggestion
	stressDefaultVMSize        = "100M"
	stressLargeVMSize          = "400M"
```

Reply:
Thanks!